BCH-Wunder-Project-team-4/Wunder.io
# Archive links

Adds archive links to frontpage resources and translations for them

Configs for translating listing paragraph headings

<img width="436" alt="Screenshot 2023-12-20 at 14 19 43" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/122397061/2c6b8949-337f-4f0f-8d63-8acc9f5888f6">

## How to test:

`git fetch origin archive-links`
`git checkout archive-links`
`lando drush cim`

Check frontpage for links buttons. Expert one is hiding :)

## Best practices:

<details>
<summary><h3>Accessibility:</h3></summary>
<p>
This project must support WCAG accessibility level AA <em>(edit this according to the requirements of your project)</em>. To ensure this standard is met, remember to:

- Perform automated checks using a tool such as Wave or SiteImprove.
- Test keyboard navigation: are all parts of the UI navigable using only the keyboard? Is the tab order logical? Can popups, menus etc be dismissed with the escape key?
- Test responsiveness, scaling and text reflow.
- Make sure no accessibility issues exist on either desktop or mobile views.
- If you have time, test with a screen reader such as VoiceOver (macOS), NVDA (Windows), or Orca (Linux).

Use the [Accessibility Testing Cheat Sheet](https://intra.wunder.io/info/accessibility-group/accessibility-testing-cheat-sheet) for information on how to run these tests.
</p>
</details>
